cni/pkg/repair/repair.go

```diff
@@ -26,10 +26,10 @@ var repairLog = scopes.CNIAgent
 
 func StartRepair(ctx context.Context, cfg config.RepairConfig) {
 	if !cfg.Enabled {
-		repairLog.Info("CNI repair is disable.")
+		repairLog.Info("CNI repair controller is disable")
 		return
 	}
-	repairLog.Info("Start CNI race condition repair.")
+	repairLog.Info("starting CNI repair controller")
```

Comment:
```suggestion
	repairLog.Info("starting CNI sidecar repair controller")
```

It would be nice if we separately labeled the sidecar vs ambient logs, but that's wishlist.

In general tho good to distinguish